ani-cli

```diff
@@ -452,6 +431,30 @@ open_episode () {
 	fi
 }
 
+play_episode () {
+	# Build command
+	set -- "$player_fn" "$video_url"
```

Comment:
I don't like the use of `set --` is there a reason you can't use a regular variable?

Reply:
I have looked into using arrays, but they are not supported under `/bin/sh` and regular variables do not vary the argument count.
On the other hand, `set --` uses the one array that all POSIX shells have. That being the arguments of the function it is called inside of.
That of course is also a downside as you need to keep in mind that each `set --` overwrites those arguments and `"$@"` contains them.

I was led to this solution by this wiki page https://github.com/koalaman/shellcheck/wiki/SC2086

Reply:
After a bit more playing around, I had realized that it can be made a bit more readable by adding:
```sh
alias add_args='set -- "$@"'
alias set_args='set --'
```
That would enable the following changes
```sh
# from
set -- "$player_fn" "$video_url"
set -- "$@" --http-referrer="$dpage_link"
# to
set_args "$player_fn" "$video_url"
add_args --http-referrer="$dpage_link"
```
I had tested this and it works.
Do you want this in the PR?

Reply:
I don't think you need an array. You can just use a simple string: `player_command="${player_fn} ${video_url}"` then add arguments like `player_command="${player_command} ${argumens}"`. Then execute it with "$player_command".

Reply:
That passes a single argument that the player will not know what to do with.
If executed without the brackets, it expands to multiple arguments, but on every space regardless of quotes

I have tried if new lines change that, but they are treated the same as spaces